pkg/controllermanager/controller/factory.go

```diff
@@ -173,7 +174,8 @@ func (f *GardenControllerFactory) Run(ctx context.Context) error {
 		}
 	}
 
-	log.Info("gardener-controller-manager initialized")
+	//nolint
+	log.Info("gardener-controller-manager initialized", "version", version.Get().GitVersion)
```

Comment:
Version is already logged in https://github.com/gardener/gardener/blob/21b1b1f3dd40936a76ed54a8d200ad55f85a19a8/cmd/gardener-controller-manager/app/gardener_controller_manager.go#L220